lib/ansible/module_utils/rabbitmq.py

```diff
@@ -193,4 +194,8 @@ def basic_publish(self):
         if args['exchange'] is None:
             args['exchange'] = ''
 
-        return self.conn_channel.basic_publish(**args)
+        try:
+            self.conn_channel.basic_publish(**args)
+            return True
```

Comment:
I don't know which code is calling this function; did you check whether that code also needs to be adjusted?

Reply:
@felixfontein - in short, no other changes required to the module.

The authors of the Pika library changed how their conn_channel.basic_publish worked.  Previously it returned errors or a tuple if successful.  They changed this to raising exceptions on errors instead, which, broke the `modules/notifications/rabbit_publish.py` module as it expected to evaluate on True or False.  Catching the success or failure here is perfectly fine as the module expects True or False.

Reply:
Perfect. Just wanted to make sure nothing got forgotten :)